code/game/machinery/doors/airlock.dm

```diff
@@ -648,23 +670,24 @@ About the new airlock wires panel:
 					H.UpdateDamageIcon()
 			else
 				visible_message("<span class='warning'>[user] headbutts the airlock. Good thing [user.p_theyre()] wearing a helmet.</span>")
-			return
-
-	if(note && ishuman(user) && user.a_intent == INTENT_GRAB)
-		user.visible_message("<span class='notice'>[user] removes [note] from [src].</span>", "<span class='notice'>You remove [note] from [src].</span>")
-		playsound(src, 'sound/items/poster_ripped.ogg', 50, 1)
-		note.forceMove(get_turf(user))
-		note = null
-		update_icon()
+			return TRUE
+	return FALSE
+
+//For the tools being used on the door. Since you don't want to call the attack_hand method if you're using hands. That would be silly
+//Also it's a bit inconsistent that when you access the panel you headbutt it. But not while crowbarring 
+//Try to interact with the panel. If the user can't it'll try activating the door
+/obj/machinery/door/airlock/proc/interact_with_panel(mob/user)
+	if(shock_user(user, 100))
 		return
-
+	// if(headbutt_airlock(user))
```

Comment:
Don't comment out code unless they have a really good reason to stay there - Just delete them